components/store/CustomerInfoFields.js

```diff
@@ -0,0 +1,221 @@
+// Module imports
+import React from 'react'
+
+
+
+
+
+// Component imports
+import ValidatedFormInput from '../ValidatedFormInput'
+import CountryTagsInput from '../CountryTagsInput'
+
+
+
+
+// Component Constants
+const INVALID_NAME_MESSAGE = 'Address is Required'
+const INVALID_EMAIL_MESSAGE = 'Address is Required'
```

Comment:
INVALID_EMAIL and INVALID_NAME both say "Address is required".

Reply:
fixed